src/unwrap.jl

```diff
@@ -300,56 +306,24 @@ end
 function get_border_range(size_img::NTuple{N, T}, border_dim, border_idx) where {N, T}
     border_range = [2:(size_img[dim]-1) for dim=1:N]
     border_range[border_dim] = border_idx:border_idx
-    return tuple(border_range...)
+    return Tuple(border_range)
```

Comment:
```suggestion
    return NTuple{N,UnitRange{Int}}(border_range)
```
(If my mental type inference worked correctly.)